src/ui/EditTask.svelte

```diff
@@ -36,6 +36,7 @@
     let parsedDueDate: string = '';
     let parsedRecurrence: string = '';
     let parsedDone: string = '';
+    let removedGlobalFilter: boolean = false;
```

Comment:
Indeed because the dialog is created every time it is opened:

```
    public onOpen(): void {
        this.titleEl.setText('Create or edit Task');
        const { contentEl } = this;
        new EditTask({
            target: contentEl,
            props: { task: this.task, onSubmit: this.onSubmit },
        });
    }
```